src/com/fundation/search/controller/ControlCriteria.java

```diff
@@ -291,39 +291,70 @@ private void listenSearchButton() {
 
         // Converting Size to bytes for Model
         if (!sizeOfSize.isEmpty()) {
-            size = converter.convertSizeStringToLong(sizeOfSize,typeOfSize);
+            size = converter.convertSizeStringToLong(sizeOfSize, typeOfSize);
         }
+        if (validateInputsGUI()) {
+            // Adding to SearchCriteria and Validating some data
+            this.searchCriteria = new SearchCriteriaBuilder()
+                    .pathCriteria(this.pathValidation(searchWindows.getPathOfCriteria()))
```

Comment:
Here we are using "pathValidation" ? and you are using "validatorPath"? difference? if you will make validation method please put all validation on this.